cf-acceptance-tests/README.md

```diff
@@ -3,7 +3,7 @@ Container for running CloudFoundry acceptance tests.
 It includes all the dependencies of the [CF test
 helpers](https://github.com/cloudfoundry-incubator/cf-test-helpers):
 
-* Go 1.5 compiler
+* Go 1.6 compiler
```

Comment:
Can we omit the version altogether? That would prevent us forgetting to update it again in the future.